.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,13 @@
+# Do not allow blank issues
+blank_issues_enabled: false
+
+contact_links:
+  - name: Microsoft Security Response Center 🔐
```

Comment:
This isn't a monarchy, this is a democracy. If @DHowett and @miniksa both think this is a good idea to add, then I'm fine with being overruled here

Reply:
How about adding a page in [MicrosoftDocs/terminal](https://github.com/MicrosoftDocs/terminal) "Reporting bugs and suggesting features" and adding a section to it "Reporting security vulnerabilities" that tells how exactly to report security vulnerabilities.

Reply:
I am talking about MSRC, btw